app/controllers/warehouse_reports/cas/chronic_reconciliation_controller.rb

```diff
@@ -2,19 +2,20 @@ module WarehouseReports::Cas
   class ChronicReconciliationController < ApplicationController
     include ArelHelper
     include WarehouseReportAuthorization
+    include SiteChronic
 
     def index
       @filter = Filter.new(filter_params)
       
-      chronic_ids = client_source.joins(:chronics).
+      chronic_ids = client_source.joins(site_chronics_sym).
         where(ch_t[:date].eq(@filter.date)).
         where(ch_t[:days_in_last_three_years].gteq(365)).
         has_homeless_service_between_dates(start_date: @filter.homeless_service_after, end_date: @filter.date).
         pluck(:id)
 
       cas_ids = client_source.cas_active.pluck(:id)
-      @missing_in_cas = client_source.joins(:chronics).
-        where(chronics: {date: @filter.date}).
+      @missing_in_cas = client_source.joins(site_chronics_sym).
+        where(site_chronics_where({date: @filter.date})).
```

Comment:
See the note about `site_chronics_table` this becomes `where(site_chronics_table => {date: @filter_date})`

Reply:
Just to confuse this even more, this is even cleaner with `merge(site_chronic_source.on_date(date: @date)`